<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>70</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>1</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>600</width>
    <height>70</height>
   </size>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>2</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="lineEdit_2" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>26</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="lineEdit" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>26</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="lblReplace" >
     <property name="sizeIncrement" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text" >
      <string>Replace with:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="lblFind" >
     <property name="sizeIncrement" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text" >
      <string>Search for:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QPushButton" name="findNext" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="font" >
      <font>
       <family>Sans Serif</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="layoutDirection" >
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text" >
      <string>&amp;Next</string>
     </property>
     <property name="icon" >
      <iconset/>
     </property>
     <property name="autoDefault" >
      <bool>true</bool>
     </property>
     <property name="default" >
      <bool>true</bool>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="replace" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="font" >
      <font>
       <family>Sans Serif</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="layoutDirection" >
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text" >
      <string>&amp;Replace</string>
     </property>
     <property name="icon" >
      <iconset/>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QPushButton" name="replaceAll" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="font" >
      <font>
       <family>Sans Serif</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text" >
      <string>Replace &amp;All</string>
     </property>
     <property name="icon" >
      <iconset/>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <widget class="QPushButton" name="findPrevious" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="font" >
      <font>
       <family>Sans Serif</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text" >
      <string>&amp;Previous</string>
     </property>
     <property name="icon" >
      <iconset/>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="0" column="4" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="font" >
      <font>
       <family>Sans Serif</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="title" >
      <string/>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>2</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item row="0" column="1" >
       <widget class="QCheckBox" name="intarget" >
        <property name="minimumSize" >
         <size>
          <width>85</width>
          <height>0</height>
         </size>
        </property>
        <property name="font" >
         <font>
          <family>Sans Serif</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>&amp;Target</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="insource" >
        <property name="minimumSize" >
         <size>
          <width>85</width>
          <height>0</height>
         </size>
        </property>
        <property name="font" >
         <font>
          <family>Sans Serif</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>&amp;Source</string>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
        <property name="tristate" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QCheckBox" name="incomment" >
        <property name="minimumSize" >
         <size>
          <width>85</width>
          <height>0</height>
         </size>
        </property>
        <property name="font" >
         <font>
          <family>Sans Serif</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>&amp;Comment</string>
        </property>
        <property name="checked" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QCheckBox" name="matchcase" >
        <property name="minimumSize" >
         <size>
          <width>85</width>
          <height>0</height>
         </size>
        </property>
        <property name="font" >
         <font>
          <family>Sans Serif</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>Matc&amp;h case</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>findNext</tabstop>
  <tabstop>findPrevious</tabstop>
  <tabstop>replace</tabstop>
  <tabstop>replaceAll</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
